nixos/modules/services/misc/navidrome.nix

```diff
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.navidrome;
+  configFile = "/etc/navidrome/settings.json";
+  dataFolder = "/var/lib/navidrome";
+in {
+  options = {
+
+    services.navidrome = {
+      enable = mkEnableOption "Enable Navidrome music server and streamer";
+
+      settings = lib.mkOption {
+        type = types.attrs;
+        default = {};
+        example = literalExample ''
+          {
+            LogLevel = "INFO";
+            BaseURL = "/music";
+            ScanInterval = "10s";
+            TranscodingCacheSize = "15MB";
+            MusicFolder = "/Media/Music";
+          };
+        '';
+        description = ''
+        Configuration for Navidrome, see <link xlink:href="https://www.navidrome.org/docs/usage/configuration-options/"/> for supported values.
+      '';
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "navidrome";
+        description = "User account under which Navidrome runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "navidrome";
+        description = "Group account under which Navidrome runs.";
+      };
+
+      musicFolderPermissions = mkOption {
```

Comment:
As someone who has never used `navidrome` I don't know why this option exists. Can you please explain why this is needed and why the user hasn't just set these permissions themselves?

Reply:
Honestly I don't think it's necessary, I started writing this derivation from the `transmission` service since it's a software I use.
Now I'm removing it, it's simpler if one simply manually sets the correct permissions.